change/@ni-nimble-components-7732a3c1-66f8-449c-bf09-eea11f258d52.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I'm not seeing a storybook build for this PR, possibly because of the (intermittent?) test failure. I'm going to wait to review until I can play with the component.

Reply:
There's a Storybook build now.